modules/dnn/src/layers/fast_convolution/winograd_3x3s1_f63.cpp

```diff
@@ -0,0 +1,1337 @@
+// Tencent is pleased to support the open source community by making ncnn available.
+//
+// Copyright (C) 2019 THL A29 Limited, a Tencent company. All rights reserved.
+//
+// Licensed under the BSD 3-Clause License (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// https://opensource.org/licenses/BSD-3-Clause
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+// This file is modified from the NCNN (https://github.com/Tencent/ncnn/blob/master/src/layer/arm/convolution_3x3_pack4.h)
+
+#include "../../precomp.hpp"
+#include "fast_convolution.hpp"
+#include "fast_convolution_internals.hpp"
+
+enum {
```

Comment:
please, move this enumeration inside cv::dnn namespace

Reply:
fixed.